sql/Pg-database.sql

```diff
@@ -1167,7 +1167,6 @@ CREATE TABLE cr_report (
     approved_by int references entity(id),
     approved_username text,
     recon_fx bool default false,
-    max_ac_id int,
```

Comment:
If you don't create the column here, you can't drop it safely in the schema migration later on (can't drop a non-existing column?).